script/playbackstrategy/modifiers/live/restartable.js

```diff
@@ -50,14 +50,10 @@ define(
         mediaPlayer.removeAllEventCallbacks();
       }
 
-      function resume () {
-        mediaPlayer.resume();
-      }
-
       function pause (opts) {
         mediaPlayer.pause();
         opts = opts || {};
-        if (opts.disableAutoResume !== true) {
+        if (opts.disableAutoResume !== true && windowType === WindowTypes.SLIDING) {
```

Comment:
No tests for this change?

Reply:
It was sliding by default. I have added one to test if it isn't a sliding window it doesn't get called.